contexts/credentials/v2

```diff
@@ -64,7 +64,58 @@
     "...": {
       "@id": "https://www.iana.org/assignments/jwt#..."
     },
-
+    "keys": {
+      "@id": "https://www.iana.org/assignments/jose#keys",
+      "@type": "@json"
+    },
+    "JsonWebKey": {
+      "@id": "https://w3id.org/security#JsonWebKey",
+      "@context": {
+        "@protected": true,
+        "id": "@id",
+        "type": "@type",
+        "controller": {
+          "@id": "https://w3id.org/security#controller",
+          "@type": "@id"
+        },
+        "publicKeyJwk": {
+          "@id": "https://w3id.org/security#publicKeyJwk",
+          "@type": "@json"
+        },
+        "secretKeyJwk": {
+          "@id": "https://w3id.org/security#secretKeyJwk",
+          "@type": "@json"
+        }
+      }
+    },
+    "Multikey": {
+      "@id": "https://w3id.org/security#Multikey",
+      "@context": {
+        "@protected": true,
+        "id": "@id",
+        "type": "@type",
+        "controller": {
+          "@id": "https://w3id.org/security#controller",
+          "@type": "@id"
+        },
+        "revoked": {
```

Comment:
I just noticed that `Multikey` supports "expires" and "revoked" but "JsonWebKey" does not.

That seems like it will cause bias  / interop issues... should we add these properties to "JsonWebKey" ?

Reply:
I'm fine with "JsonWebKey" adopting "expires" and "revoked". Or not -- depending on what people think makes the most sense for users of the two approaches.

That being said, I do not understand how you have been using the word "bias" in a number of places. I can only make sense of your use of it by thinking you mean that if two things are not exactly the same then one may favor one over the other for a variety of reasons. But I don't find that definition of "bias" to identify something that ought to be (or even practically can be) avoided. Nor is that definition what I think most people think of when they use the word. It's not what I mean when I say "bias". I think most people think of "bias" as "undue influence" or "lack of fairness".

A securing mechanism that doesn't change the format of the input has advantages. Does this mean that all securing mechanisms that do change the format don't have their own advantages and should be changed?

A spec that allows you to choose between two different key formats has the advantage of choice; another spec that recommends a particular one has the advantage of simplicity. These are not the same; should it be impossible to have both of these at the same time?

I choose these examples because I don't believe you've raised them as potentially causing "bias issues", but it feels like I'm applying the same principle you seem to apply here and elsewhere. I don't agree with that principle. I think the principle we should be applying is related to fairness, not difference.

While a difference is required in order for something to be unfair, clearly something just being different doesn't mean there is a lack of fairness or undue influence. Otherwise everything in every spec we produce would have to be exactly the same to eradicate this unfairness. Taken to its logical end, to remove that kind of "bias" we would have to eliminate all specs since defining anything creates a distinction between what matches the definition and what does not. So I presume this can't be what you mean by "bias" and the desire to eliminate it, but then I haven't been able to think of any consistent definition for what you may mean.

Perhaps you think there's something unfair about one key format being able to express X and another not being able to. But I don't understand that either. I think there are simply trade offs -- and it's those trade offs that should inform whether it makes sense (or not) to adopt certain features in any format. If you think "JsonWebKey" should do X, or Y, or Z, please just say so and see if others agree or disagree. Raising the specter of "bias" seems unnecessary and confusing.

Reply:
I'm fine w/ `JsonWebKey` supporting the `revoked` property. Suggest adding it to align those two. I will do a similar PR in the Data Integrity spec.

Reply:
PR raised in https://github.com/w3c/vc-data-integrity/pull/184